lib/Parse/ParseExpr.cpp

```diff
@@ -1207,9 +1207,9 @@ Parser::parseExprPostfixSuffix(ParserResult<Expr> Result, bool isExprBasic,
       if (canParseAsGenericArgumentList()) {
         SmallVector<TypeRepr *, 8> args;
         SourceLoc LAngleLoc, RAngleLoc;
-        if (parseGenericArguments(args, LAngleLoc, RAngleLoc)) {
+        auto argStat = parseGenericArguments(args, LAngleLoc, RAngleLoc);
```

Comment:
Shouldn't this status be incorporated into the ParserResult about to be returned?

Reply:
Currently, that's unreachable because `canParseAsGenericArgumentList()` fails if CC token is in it.